modules/core_api/fsw/inc/cfe_version.h

```diff
@@ -21,41 +21,40 @@
 /**
  * @file
  *
- * Provide version identifiers for the cFE core.
+ * Provide version identifiers for the cFE core. See @ref cfsversions for further details.
  */
 
 #ifndef CFE_VERSION_H
 #define CFE_VERSION_H
 
 /* Development Build Macro Definitions */
-#define CFE_BUILD_NUMBER 593 /*!< Development Build: Number of commits since baseline */
-#define CFE_BUILD_BASELINE                                                                    \
-    "v6.8.0-rc1" /*!< Development Build: git tag that is the base for the current development \
-                  */
-
-/* Version Macro Definitions */
-#define CFE_MAJOR_VERSION 6 /*!< @brief ONLY APPLY for OFFICIAL releases. Major version number. */
-#define CFE_MINOR_VERSION 7 /*!< @brief ONLY APPLY for OFFICIAL releases. Minor version number. */
-#define CFE_REVISION                                                                                                  \
-    99 /*!< @brief ONLY APPLY for OFFICIAL releases. Revision version number. A value of "99" indicates a development \
-          version.  */
-#define CFE_MISSION_REV 0 /*!< @brief ONLY USED by MISSION Implementations. Mission revision */
-
-#define CFE_STR_HELPER(x) #x                /*!< @brief Helper function to concatenate strings from integer macros */
-#define CFE_STR(x)        CFE_STR_HELPER(x) /*!< @brief Helper function to concatenate strings from integer macros */
-
-/*! @brief Development Build Version Number.
- * @details Baseline git tag + Number of commits since baseline. @n
- * See @ref cfsversions for format differences between development and release versions.
+#define CFE_BUILD_NUMBER   593          /**< @brief Development: Number of development commits since baseline */
+#define CFE_BUILD_BASELINE "v6.8.0-rc1" /**< @brief Development: Reference git tag for build number */
+
+/* Version Macro Definitions updated for official releases only */
+#define CFE_MAJOR_VERSION 6  /**< @brief Major release version (Former for Revision == 99) */
+#define CFE_MINOR_VERSION 7  /**< @brief Minor release version (Former for Revision == 99) */
+#define CFE_REVISION      99 /**< @brief Revision, 99 indicates development */
+#define CFE_MISSION_REV   0  /**< @brief Mission revision, reserved for mission use */
+
+#define CFE_STR_HELPER(x) #x                /**< @brief Convert agrument to string */
```

Comment:
we should probably use something other than "x" here any ideas?

Reply:
'y'?  J/k... it's a generic parameter, so 'x' is good enough for me.  Feel free to suggest something different though.